src/foundry/client/data/documents/active-effect.d.mts

```diff
@@ -18,22 +22,45 @@ declare global {
    */
   class ActiveEffect extends ClientDocumentMixin(foundry.documents.BaseActiveEffect) {
     /**
-     * Is there some system logic that makes this active effect ineligible for application?
-     * @defaultValue `false`
+     * Create an ActiveEffect instance from some status effect ID.
+     * Delegates to {@link ActiveEffect._fromStatusEffect} to create the ActiveEffect instance
+     * after creating the ActiveEffect data from the status effect data if `CONFIG.statusEffects`.
+     * @param statusId - The status effect ID.
+     * @param options  - Additional options to pass to the ActiveEffect constructor.
+     * @returns The created ActiveEffect instance.
+     *
+     * @throws An error if there is no status effect in `CONFIG.statusEffects` with the given status ID and if
+     * the status has implicit statuses but doesn't have a static _id.
      */
-    get isSuppressed(): boolean;
+    static fromStatusEffect(statusId: string, options?: DocumentConstructionContext | undefined): Promise<ActiveEffect>;
```

Comment:
No need for the `| undefined`; that's just for object properties. Functions don't get treated any differently at the type level. Don't take it up with me.